controllers/statusreport/statusreport_adapter.go

```diff
@@ -74,9 +77,7 @@ func (a *Adapter) EnsureSnapshotTestStatusReportedToGitHub() (controller.Operati
 
 	for _, reporter := range reporters {
 		if err := reporter.ReportStatusForSnapshot(a.client, a.context, &a.logger, a.snapshot); err != nil {
-			a.logger.Error(err, "failed to report test status to github for snapshot",
```

Comment:
I think it would be good to keep the log error here so we can pick up on the issue when debugging.